server/server.js

```diff
@@ -90,6 +91,16 @@ app.route('GET', '/bundle.css', function (req, res, ctx) {
   })
 })
 
+// Serve static files for the Stencila UIs from here rather than from
+// inside session containers
+app.route('GET', '/static/stencila/*', function (req, res, ctx) {
+  const pathname = url.parse(req.url).pathname
+  const source = send(req, path.join('node_modules/stencila/build/', pathname.substring(17)))
```

Comment:
using `substring()` seems little brittle, but given we're already using it in places and we got a deadline it might be alright.